cves/CVE-2015-1831.yml

```
@@ -144,10 +146,10 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: This commit was interestion because they seemed to know that the excludeParams wasn't working as intended but the way they went about to add addtional exluded patterns was wrong.
```

```suggestion
  answer: This commit was interesting because they seemed to know that the excludeParams wasn't working as intended but the way they went about adding additional excluded patterns was wrong.
```
I'd suggest running your answers through a simple spelling/grammar checker; I found a lot of small mistakes like this throughout your PR and it made it difficult for me to understand and pay attention to your research.

---

Went through and fixed my spelling.